packages/ui-react/src/pages/ScreenRouting/PlaylistScreenRouter.tsx

```diff
@@ -2,27 +2,26 @@ import React from 'react';
 import { useParams } from 'react-router';
 import { useTranslation } from 'react-i18next';
 import type { Playlist } from '@jwp/ott-common/types/playlist';
-import { CONTENT_TYPE } from '@jwp/ott-common/src/constants';
+import { PLAYLIST_TYPE, PLAYLIST_CONTENT_TYPE } from '@jwp/ott-common/src/constants';
 import { ScreenMap } from '@jwp/ott-common/src/utils/ScreenMap';
 import usePlaylist from '@jwp/ott-hooks-react/src/usePlaylist';
 
 import Loading from '../Loading/Loading';
 import ErrorPage from '../../components/ErrorPage/ErrorPage';
 import type { ScreenComponent } from '../../../types/screens';
+import useQueryParam from '../../hooks/useQueryParam';
 
 import PlaylistGrid from './playlistScreens/PlaylistGrid/PlaylistGrid';
 import PlaylistLiveChannels from './playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels';
 
 export const playlistScreenMap = new ScreenMap<Playlist, ScreenComponent<Playlist>>();
 
-// register playlist screens
-playlistScreenMap.registerByContentType(PlaylistLiveChannels, CONTENT_TYPE.live);
-playlistScreenMap.registerDefault(PlaylistGrid);
-
 const PlaylistScreenRouter = () => {
   const params = useParams();
   const id = params.id || '';
-  const { isLoading, isFetching, error, data } = usePlaylist(id);
+  const type = useQueryParam('t') === 'c' ? PLAYLIST_TYPE.content_list : PLAYLIST_TYPE.playlist;
+
```

Comment:
Hmm, I have doubts about using query params for this. Do we include this in the canonical URL?

Alternatively, we could do this:

```tsx
export const PATH_PLAYLIST_CONTENT = '/p/:id/c/*' // or something similar

// ...

<Route path={PATH_PLAYLIST_CONTENT} element={<PlaylistScreenRouter type="content" />} />
<Route path={PATH_PLAYLIST} element={<PlaylistScreenRouter type="playlist" />} />
```

Reply:
I think we would prefer to avoid changing playlist routing, so I was choosing between a separate screen routing and a query param. If we change the path, then I think we should also change it for the playlist.

Reply:
Probably safest to keep them as separate paths. Why not just `/p/:id/*` and `/n/:id/*` (using n for coNtent list, because we might want to reserve c for individual content items if we ever get to that refactor.) I worry `/p/:id/c/*` will have collisions with `/p/:id/*` if we aren't careful.

Reply:
That works for me 😄